src/Swift.Runtime/src/Swift/Runtime/ProtocolConformanceDescriptor.cs

```diff
@@ -141,4 +155,10 @@ public static ProtocolConformanceDescriptor LoadFromSymbol(string libraryName, s
             NativeLibrary.Free(libraryHandle);
         }
     }
+
+    private static ProtocolConformanceDescriptor GetProtocolConformanceDescriptor(TypeMetadata metadata, ProtocolDescriptor descriptor)
+        => swift_conformsToProtocol(metadata, descriptor);
+
+    [DllImport(KnownLibraries.SwiftCore, CallingConvention = CallingConvention.Cdecl)]
+    private static extern ProtocolConformanceDescriptor swift_conformsToProtocol(TypeMetadata metadata, ProtocolDescriptor descriptor);
```

Comment:
Is this a the function https://github.com/swiftlang/swift/blob/136cdee1950a5e72b99d7b8177bb2ab5043cd974/include/swift/Runtime/RuntimeFunctions.def#L1863?

I think it returns a `ProtocolWitnessTable` not a protocol conformance descriptor. Which makes me wonder how it works. But on the other hand we don't have any tests yet which verify that.

Reply:
Yes, I missed that. Let's move it to the PWT and add tests.

Reply:
These changes will be introduced in a separate PR.